### PR TITLE
Update DG and UML diagram

### LINKED ISSUE #75: Update UML Diagram and DG
Update DeleteSequenceDiagram to match code changes 

Update Developer Guide for Delete Command

### PR DESCRIPTION
**Pull request by The0nlyJuan**
Update DG to for changes to Delete Command

Update UML diagram for DeleteSequenceDIagram

Closes #75

### CODE REVIEW

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 1]
```diff
@@ -49,21 +49,39 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
-DeleteCommand -> Model : deletePerson(1)
-activate Model
+alt Person not found
+    DeleteCommand -> LogicManager : throw CommandException
+    LogicManager -> MainWindow : showError("Invalid person name")
```
Comment (SSirMentos) [inline docs/diagrams/DeleteSequenceDiagram.puml]: Do we have to include MainWindow?

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 2]
```diff
@@ -49,21 +49,39 @@ deactivate AddressBookParser
...
 
-Model --> DeleteCommand
-deactivate Model
+        Model --> DeleteCommand
+        deactivate Model
 
-create CommandResult
-DeleteCommand -> CommandResult
-activate CommandResult
+        DeleteCommand -> LogicManager : return CommandResult("Duplicate persons found. Please delete by index.")
```
Comment (SSirMentos) [inline docs/diagrams/DeleteSequenceDiagram.puml]: omit the return?

**Review by SSirMentos — COMMENTED** [review]
LGTM

**Review by Edwin1022 — COMMENTED** [review]
Looks good to me!!

**Pull request merged**